Image width and height modification

AlexanderSk opened the pull request:
Updated for correct image tampering within the dom.
Function recursive_Array_Search added.
also preg_match is modified once again in order to include only
specified extensions.

Note: perhaps it would be better to place array search in the helper?
The changes are not tested against live version websites with no issues.
Also note that php 5.x can support multi dimensional array search but for compatibility i placed in this function (source php.net).

AlexanderSk committed:
Image width and height modification

Updated for correct image tampering within the dom.
Function recursive_Array_Search added.
also preg_match is modified once again in order to include only
specified extensions.

AlexanderSk commented:
Please hold with this Pull request.
I noticed that the css file is not generated after saving the dom->html with this implementation.
Will check it further tomorrow.

yireo commented:
Ok, let me know once you want me to bring in the changes. Great changes so far and keep up the good work!

AlexanderSk committed:
Merge remote-tracking branch 'refs/remotes/yireo/master'

AlexanderSk committed:
Revert "Image width and height modification"

This reverts commit 521db41894fc27a4c2ee3ebbf1b5eacdd2ddbd5a.

AlexanderSk committed:
Minor change

The images should only be the type our library can support in order to
exclude any unforeseen issues. in case standars change we will update
the code as needed.
